src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -0,0 +1,169 @@
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import org.mindrot.jbcrypt.BCrypt;
+import java.util.UUID;
+import java.time.Instant;
+
+
+/** Servlet class responsible for the Admin page. */
+public class AdminServlet extends HttpServlet {
+
+  private UserStore userStore;
+  private ConversationStore conversationStore;
+  private MessageStore messageStore;
+  private List<String> nameOfUserWhoWantToBeAdmin = new ArrayList<>();
+  /**
+   * Set up state for handling login-related requests. This method is only called when running in a
+   * server, not when running in a test.
+   */
+
+  @Override
+  public void init() throws ServletException {
+    super.init();
+    setUserStore(userStore.getInstance());
+    setConversationStore(conversationStore.getInstance());
+    setMessageStore(messageStore.getInstance());
+
+  }
+
+  /**
+   * Sets the UserStore used by this servlet. This function provides a common setup method for use
+   * by the test framework or the servlet's init() function.
+   */
+  void setUserStore(UserStore userStore) {
+    this.userStore = userStore;
+  }
+  /**
+   * Sets the ConversationStore used by this servlet. This function provides a common setup method for use
+   * by the test framework or the servlet's init() function.
+   */
+  void setConversationStore(ConversationStore conversationStore) {
+    this.conversationStore = conversationStore;
+  }
+
+  /**
+   * Sets the MessageStore used by this servlet. This function provides a common setup method for
+   * use by the test framework or the servlet's init() function.
+   */
+  void setMessageStore(MessageStore messageStore) {
+    this.messageStore = messageStore;
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws IOException, ServletException {
+      String username = (String)request.getSession().getAttribute("user");
+
+      if(username == null) {
+        // user is not logged in, don't let them access the admin page
+        response.sendRedirect("/");
+        return;
+      }
+      List<String> adminUsernames = new ArrayList<String>();
+      adminUsernames.add("admin");
+      adminUsernames.add("admin1");
+      adminUsernames.add("admin2");
+      adminUsernames.add("admin3");
+
+
+      User user = userStore.getUser(username);
+
+      if (adminUsernames.contains(username)){
+        User userAdmin = userStore.getUser(username);
+        user.setIsAdmin(true);
+      }
+
+      if(user == null) {
+        // user was not found, don't let them access the admin page
+        response.sendRedirect("/");
+        return;
+      }
+      //user.setIsAdmin(true);
+      if(user.getIsAdmin() == false) {
+        response.sendRedirect("/");
+        return;
+      }
+      /* an attempt to grab information from the stores to display on the page
+      if the user is admin*/
+
+        int numOfUsers = UserStore.getInstance().amountOfUsers();
```

Comment:
You already have instances of all the store objects. Also, for the UserStore you're using the class rather than the object you created.